### PR TITLE
Trim fix: This allows to use the RC trim button to copy trims in all flight modes

### PR DESCRIPTION
**Pull request by LorenzMeier**
This is intensely bench tested, but needs flight testing. It also mandates QGC master branch for the button to work. Setting TRIM_ROLL, TRIM_PITCH, TRIM_YAW on the command line should work as well. This PR should affect all fixed wing flight modes and allow to trim an airframe with the RC.

@kd0aij I would deeply appreciate if you could test this.

### CODE REVIEW

File: src/modules/fw_att_control/fw_att_control_main.cpp [hunk 1]
```diff
@@ -1081,9 +1078,9 @@ FixedwingAttitudeControl::task_main()
 
 			} else {
 				/* manual/direct control */
-				_actuators.control[0] = _manual.y;
-				_actuators.control[1] = -_manual.x;
-				_actuators.control[2] = _manual.r;
+				_actuators.control[0] = _manual.y + _parameters.trim_roll;
+				_actuators.control[1] = -_manual.x + _parameters.trim_pitch;
```
Comment (tumbili) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: @LorenzMeier minus here?

Comment (tumbili) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: I'm confused. I change the sign of the pitch trim param in the code and nothing changes.

**Comment by LorenzMeier** [comment]
Awesome, thanks!

**Comment by kd0aij** [comment]
Tried this in HIL with latest release_v1.0.0. 
More preflight check failure messages on entering the Radio Calibration page :-1: 
The TRIM_ROLL, TRIM_PITCH parameters changed on pressing the "Copy trims" button; the units aren't specified in the metadata, but it looks like normalized from -1 to 1. :+1: 

Reflashed my Ninox and repeated the procedure to capture the trims from my last test flight... TRIM parameters stayed at zero. :-1: 
FW git-hash: e94bc7960c027d8e806e026b3c25d79c378a6fe7

**Comment by LorenzMeier** [comment]
What are the manual control values in QGC, or in other words, which values should we be seeing in the params? Any text messages about a trim cal fail?

**Comment by kd0aij** [comment]
RC1,2 are 1512, 1476
no messages on nsh console or in QGC

**Comment by LorenzMeier** [comment]
What are MANUAL_CONTROL.x and MANUAL_CONTROL.y? They have a deadzone applied to them. It might be that the IO Firmware doesn't have the deadzone, which would explain the difference.

**Comment by kd0aij** [comment]
MANUAL_CONTROL.x and MANUAL_CONTROL.y neutral points are at 131, 11 with trim.
It would not be good to have a deadband applied to trim values.

**Comment by LorenzMeier** [comment]
I can't reproduce your issue. I get the correct behaviour. Could you re-test with a larger stick setting to work out if you're just seeing the deadzone? That would be a different discussion then, where we can see if we want to default the deadzone to zero. Right now its +- 10, so 0.02 / 2% of the total stick motion by default.

If you still get zero trim values you can try setting the RC_*_DZ values to zero and see if things behave after a reboot.

**Comment by kd0aij** [comment]
Restarting QGC fixed it. Either I had something hung up, or I managed to run the wrong version of QGC between my HIL test and the Ninox.
:+1: TRIM parameters set and TX trims rezeroed. Now waiting for decent flying weather.

**Pull request merged**